app/controllers/api/v1/events_by_month_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class EventsByMonthController < Api::V1::ApiController
```

Comment:
A single resource of `EventsCalendar` that only has a `show` method like `Feed`might be a better name, or `MonthsEvents` that receives the month as part of the url

Reply:
Something like this? 
`class EventsCalendarController < Api::V1::ApiController
      def show
        date = params[:id] ? Time.zone.parse(params[:id]) : Time.zone.now`
To be called: **api/v1/events_calendar/2020-11-01**